### ISSUE TITLE
Nothing is created with new DokuWiki release rc2020-06-01 "Hogfather"

### ISSUE THREAD

**Issue by joerg-nafcom** [gkrid/dokuwiki-plugin-bez#8]
Hi there. We would love to use your bez plugin to manage tasks within our team. I installed all DB plugins as required according to doc page at https://www.dokuwiki.org/plugin:bez however it appears with the recently released new DokuWiki release rc2020-06-01 "Hogfather", nothing is created, once I try to create project, problem or task! Once I filled out the forms and hit Save, nothing is actually appearing to be saved.

**Comment by solewniczak** [comment]
I'll check it when my company migrates to Hogfather. We are waiting for the stable version (not the release candidate).

**Comment by joerg-nafcom** [comment]
Thanks for your reply! Can you please let me know here when you had a look and the issue (bug) has been resolved so we know when we can try again? Thanks in advance! :)

**Comment by solewniczak** [comment]
Yes of course :) As I said, we are waiting for the stable release.

**Comment by joerg-nafcom** [comment]
thanks! I saw you have regardless released a new update yesterday, but there is no change log what has been modified, it anyway still doesn't work for me

**Comment by solewniczak** [comment]
This was probably a problem with JavaScript. I've fixed it now and the last version of bez plugin should work with Hogfather. Please check it out.
